ledger/object/record_db.go

```diff
@@ -43,6 +44,11 @@ func (k recordKey) Scope() store.Scope {
 	return store.ScopeRecord
 }
 
+func (k recordKey) String() string {
```

Comment:
Why do we need it? If it uses a `DebugString` inside, it shold be called `DebugString` as well.

Reply:
Because in code I have to do smh like that: 
kkk := insolar.ID(key)
kkk.DebugString()

Reply:
Two times.
It doesn't look good

Reply:
Didn't agree. @LinnikD your decision.

Reply:
They are the same for me